sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -94,4 +112,24 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         );
     }
   }
+
+  /**
+   * Updates the infected file status and renames file to describe an error with the file.
+   * @param studentFile student infected file to be updated.
+   * @returns the update result.
+   */
+  private async updateInfectedFile(
+    studentFile: StudentFile,
+  ): Promise<UpdateResult> {
+    const fileName = path.parse(studentFile.fileName).name;
+    const fileExtension = path.parse(studentFile.fileName).ext;
+
+    return this.repo.update(
+      { id: studentFile.id },
+      {
+        fileName: `${fileName}${INFECTED_FILENAME_SUFFIX}${fileExtension}`,
+        virusScanStatus: VirusScanStatus.VirusDetected,
```

Comment:
Please add the audit fields.
I noticed they are missing also in the other saving operation, if you don't mind, please add them there also.